exchanges/lbank/lbank_wrapper.go

```diff
@@ -966,6 +967,12 @@ func (l *Lbank) GetHistoricCandlesExtended(pair currency.Pair, a asset.Item, sta
 		}
 	}
 
+	err = dates.Verify(ret.Candles)
+	if err != nil {
+		log.Warn(log.ExchangeSys, err.Error())
```

Comment:
Just a thought: Might be better to return an error if any of the candles are invalid?

Reply:
The only error that is returned is whether it is missing data at certain intervals. I figure it was better to just warn that the API/exchange itself is not returning all the data you request, there isn't much that can be done in such a scenario. I'll try to think of how it can be reworked

Reply:
That's fair! Yeah, I don't know what to suggest other than changing the name so it's a bit more applicable. I would expect it to spam a little bit on very low liquid exchanges/ illiquid pairs at lower time intervals.